test/interval_tests/numeric.jl

```diff
@@ -197,3 +197,17 @@ end
 
     setrounding(Interval, :narrow)
 end
+
+@testset "Fast power" begin
+    x = 1..2
+    @test pow(x, 2) == pow(-x, 2) == Interval(1, 4)
+    @test pow(-x, 3) == Interval(-8.0, -1.0)
+
+    @test pow(-1..2, 2) == 0..4
+    @test pow(-1..2, 3) == -1..8
+    @test pow(-1..2, 4) == 0..16
+
+    x = @interval(pi)
+    @test x^100 ⊆ pow(x, 100)
```

Comment:
I do like this test! Yet, I think this should be extende to include tests which play a little bit with the rounding or with other `Interval{T}` types. 

It would also be nice to have an example where `interior(a^n, pow(a,n))` is `true`.